Jenkinsfile

```diff
@@ -27,42 +27,47 @@ def getChangeDetail() {
     return changeDetail
 }
 
+IS_MASTER_BRANCH = env.BRANCH_NAME == "master"
+IS_RELEASE_BRANCH = (env.BRANCH_NAME ==~ /\d+\.\d+\.\d+/)
+
 def sendEmailNotification() {    
+    if (!(IS_MASTER_BRANCH || IS_RELEASE_BRANCH)) {
```

Comment:
thinking about it, I don't think I want this check at all. If it's too many emails, I can always add it later, but I definitely want PR emails.